app/src/main/java/io/homeassistant/companion/android/sensors/PhoneStateSensorManager.kt

```diff
@@ -14,13 +17,25 @@ class PhoneStateSensorManager : SensorManager {
             "sensor",
             "Phone State"
         )
+
+        val sim_1 = SensorManager.BasicSensor(
+            "sim_1",
+            "sensor",
+            "SIM_1"
+        )
+
+        val sim_2 = SensorManager.BasicSensor(
+            "sim_2",
+            "sensor",
+            "SIM_2"
+        )
     }
 
     override val name: String
         get() = "Phone Sensors"
 
     override val availableSensors: List<SensorManager.BasicSensor>
-        get() = listOf(phoneState)
+        get() = listOf(phoneState, sim_1, sim_2)
```

Comment:
We should check the version here and only add `sim_1` and `sim_2` if the version of android supports them.  Otherwise they will show up for older versions but crash the app if they try to enable!

Reply:
Yes, my bad